elixir.go

```diff
@@ -126,24 +129,31 @@ func generateJwtFromElixir(idStruct ElixirIdentity, key, alg string) (string, er
 		log.Error("Claims in token are empty")
 	}
 
+	u, err := url.Parse(iss)
+		if err != nil {
+		return "", fmt.Errorf("failed to pars ISS, reason: %v", err)
```

Comment:
there was a typo, so I made the message clearer for those of us who always forget the fields ^^
```suggestion
	return "", fmt.Errorf("failed to parse ISS (jwt issuer field), reason: %v", err)
```